src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/Constants.java

```diff
@@ -0,0 +1,9 @@
+package uk.nhs.digital.nhsconnect.lab.results.model;
+
+@SuppressWarnings("checkstyle:hideutilityclassconstructor")
+public final class Constants {
```

Comment:
Maybe this is `SystemConstants`?

Reply:
Nah `Constants` is fine. This kind of file is common practice across Java projects